onadata/libs/templates/account_lockout/lockout_start.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans with lockout_time=lockout_time username=username %}
+Hi {{ username }},
+
+You have been locked out from your Ona account because of too many wrong passwords.
+Please wait for {{ lockout_time }} minutes before trying to access your account again.
+If this was not you please reach out to our support team for help.
+Thank you for choosing Ona!
+
+Thanks,
+The Team at Ona
```

Comment:
I think there should be line spacing in between the lines, i.e:
```TEXT
Hello {{ username}}

You have been locked out from your Ona account because of too many wrong passwords.

Please wait for {{ lockout_time }} minutes before trying to access your account again.

If this was not you please reach out to our support team for help.

Thank you for choosing Ona!
```

The same should also apply to the other email.